conjur-env/go.mod

```diff
@@ -2,10 +2,12 @@ module github.com/cyberark/cloudfoundry-conjur-buildpack/conjur-env
 
 require (
 	github.com/cyberark/conjur-api-go v0.6.0
-	github.com/cyberark/summon v0.8.0
-	github.com/sirupsen/logrus v1.4.2 // indirect
-	golang.org/x/sys v0.0.0-20200124204421-9fbb57f87de9 // indirect
-	gopkg.in/yaml.v2 v2.2.8 // indirect
+	github.com/cyberark/summon v0.8.2
```

Comment:
fantastic! can you please also update the NOTICES in this PR? it will make it easier for when we release

I _think_ based on my more recent understanding of modules, that we can exclude the `indirect` deps from the NOTICES (since we document first-tier, manually included deps only)

when you make the updates, please verify that the copyright snippets are still valid for each dep.

Reply:
It looks like the only change was related to the summon version, so I added that to the PR 👍